### PR TITLE
chore: substitute calls to logging functions using testing.T with std…

### PR DESCRIPTION
**Pull request by UtheMan**
…lib log package

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: e2e/suite_test.go [hunk 1]
```diff
@@ -147,11 +150,11 @@ func bootstrapVMSS(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scena
 	}
 
 	vmssName := fmt.Sprintf("abtest%s", randomLowercaseString(r, 4))
-	t.Logf("vmss name: %q", vmssName)
+	log.Printf("vmss name: %q", vmssName)
 
 	cleanupVMSS := func() {
-		t.Log("deleting vmss", vmssName)
-		poller, err := opts.cloud.vmssClient.BeginDelete(ctx, *opts.chosenCluster.Properties.NodeResourceGroup, vmssName, nil)
+		log.Println("deleting vmss", vmssName)
```
Comment (cameronmeissner) [inline e2e/suite_test.go:156]: can we be consistent on Println with multiple args vs. Printf? I'd personally lean towards Printf but that's just me

File: e2e/suite_test.go [hunk 2]
```diff
@@ -160,7 +163,7 @@ func bootstrapVMSS(ctx context.Context, t *testing.T, r *mrand.Rand, opts *scena
 		if err != nil {
 			t.Error("error polling deleting vmss", vmssName, err)
 		}
-		t.Logf("finished deleting vmss %q", vmssName)
+		log.Printf"finished deleting vmss %q", vmssName)
```
Comment (cameronmeissner) [inline e2e/suite_test.go:166]: seems like a syntax error somewhere around here caused the latest run to fail

File: e2e/suite_test.go (hunk 1 above)
Comment (UtheMan) [reply e2e/suite_test.go]: fixed, switched log.Println calls where there were extra args to log.Printf

File: e2e/suite_test.go (hunk 2 above)
Comment (UtheMan) [reply e2e/suite_test.go]: yeah missing parenthesis :'(

**Pull request closed without merging**